catalog/dags/database/batched_update/batched_update_dag.py

```diff
@@ -181,6 +162,9 @@
     },
 )
 def batched_update():
+    # Unique Airflow variable name for tracking the batch_start for this query
+    BATCH_START_VAR = "batch_start_{{ params.query_id }}"
```

Comment:
Would it be possible to have this match the DAG ID so it's potentially a bit easier to identify? Maybe:
```suggestion
    BATCH_START_VAR = "batched_update_start_{{ params.query_id }}"
```